source/blog/2017-05-25-future-plans-for-fedora-atomic-release-life-cycle.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Future Plans for the Fedora Atomic Release Life Cycle
+author: jbrooks
+date: 2017-05-25 07:00:00 UTC
+tags: fedora
+comments: true
+published: true
+---
+
+The Fedora Project ships new releases at roughly six-month intervals, and [maintains](https://fedoraproject.org/wiki/Fedora_Release_Life_Cycle) each release for around thirteen months. So Fedora N is supported by the community until one month after the release of Fedora N+2.
+
+Since the first Fedora Atomic host shipped, as part of Fedora 21, the project has maintained separate ostree repositories for both of the active Fedora releases. For instance, there are currently trees available for Fedora Atomic 25 and Fedora Atomic 24.
+
+Fedora Atomic sets out to be a particularly fast-moving branch of Fedora, with releases every two weeks and updates to key “atomic” components such as docker and kubernetes that move more quickly than one might expect from Fedora.
+
+Due in part to this faster pace, the Fedora Atomic workgroup has always focused its testing and integration efforts most directly on the latest stable release, encouraging users of the older release to rebase to the newer tree as soon as possible. Releases older than the current tree are supported only on a "best effort" basis, meaning that the ostree is updated, but there is no organized testing of the older releases.
+
+Starting with either the Fedora 26 to 27 or the 27 to 28 upgrade cycle, the Fedora Atomic Workgroup intends to collapse Fedora Atomic into a single version which will track the latest stable Fedora branch. When a new stable version of Fedora is released, Fedora Atomic users will automatically shift to the new version when they install updates.
```

Comment:
Your comment is about this detail, so if it's unimportant, we can ignore your comment. The whole point of this post is to say that in the future, we're moving to a single atomic host stream, and that until we do that, users should only expect best-effort support for N-1. After the change, there's no question of best-effort support, because there will be no old release to support.

The symlink thing was one proposed way to do it in the short term -- you could rebase to the symlinked ref, and have the rolling behavior, even before we actually switched to  a rolling release. I refer to this half measure in the second to last graf of this post.

If the sig isn't ready to go rolling in the 26 to 27 time frame, we can make that switch in the 27 to 28 time frame, and the post already mentions that this might be the timing.

Look, we can hold off on this post and re-discuss, but I'm quite sure that "they will have the option to follow a ref that is not latest" was never part of the plan.